configs/gcvit/README.md

```diff
@@ -0,0 +1 @@
+TBD
```

Comment:
Please update this file as other models, you can copy a readme file from one model in configs